devtools/pixi-envs/pyproject.toml

```diff
@@ -0,0 +1,11 @@
+[project]
+name = "openff-utilities"
+dependencies = [ "mypy==1.6", "pytest-cov>=5.0.0,<6", "types-setuptools>=75.2.0.20241019,<76", "types-toml>=0.10.8.20240310,<0.11", "pytest", "importlib-resources"]
+
+[tool.pixi.project]
+channels = ["conda-forge", "openeye"]
```

Comment:
For me: might not need `openeye` channel if it can be exposed when the package is installed

Reply:
@mattwthompson AFAIK you cannot add the channel alongside an dependency installation. You need a separate command for this I think 🤔

Reply:
Hmm yeah, looks like that's the design here

Can update the project config with a new channel: https://prefix.dev/docs/pixi/cli#project-channel-add

But not when installing a package into an existing environment: https://prefix.dev/docs/pixi/cli#add